Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Internal Server Error: /marketing_page/
Traceback (most recent call last):
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\Downloads\snape\SocialMediaAnalysisPlatform\snape\myproject\main\views.py", line 658, in marketing_page
    return render(request, 'main/marketing_page.html' ,
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 106, in render
    url = self.url(context)
          ^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 103, in url
    return self.handle_simple(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 118, in handle_simple
    return staticfiles_storage.url(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 147, in url
    return self._url(self.stored_name, name, force)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 126, in _url
    hashed_name = hashed_name_func(*args)
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 417, in stored_name
    raise ValueError("Missing staticfiles manifest entry for '%s'" % clean_name)
ValueError: Missing staticfiles manifest entry for 'assets/snap.mp4'
"GET /marketing_page/ HTTP/1.1" 500 145
Internal Server Error: /marketing_page/
Traceback (most recent call last):
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\Downloads\snape\SocialMediaAnalysisPlatform\snape\myproject\main\views.py", line 658, in marketing_page
    return render(request, 'main/marketing_page.html' ,
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 106, in render
    url = self.url(context)
          ^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 103, in url
    return self.handle_simple(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 118, in handle_simple
    return staticfiles_storage.url(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 147, in url
    return self._url(self.stored_name, name, force)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 126, in _url
    hashed_name = hashed_name_func(*args)
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 417, in stored_name
    raise ValueError("Missing staticfiles manifest entry for '%s'" % clean_name)
ValueError: Missing staticfiles manifest entry for 'assets/snap.mp4'
"GET /marketing_page/ HTTP/1.1" 500 145
Internal Server Error: /marketing_page/
Traceback (most recent call last):
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\Downloads\snape\SocialMediaAnalysisPlatform\snape\myproject\main\views.py", line 658, in marketing_page
    return render(request, 'main/marketing_page.html' ,
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\defaulttags.py", line 211, in render
    nodelist.append(node.render_annotated(context))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 106, in render
    url = self.url(context)
          ^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 103, in url
    return self.handle_simple(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 118, in handle_simple
    return staticfiles_storage.url(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 147, in url
    return self._url(self.stored_name, name, force)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 126, in _url
    hashed_name = hashed_name_func(*args)
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 417, in stored_name
    raise ValueError("Missing staticfiles manifest entry for '%s'" % clean_name)
ValueError: Missing staticfiles manifest entry for '/assets/profile.jpg'
"GET /marketing_page/ HTTP/1.1" 500 145
Internal Server Error: /marketing_page/
Traceback (most recent call last):
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\Downloads\snape\SocialMediaAnalysisPlatform\snape\myproject\main\views.py", line 658, in marketing_page
    return render(request, 'main/marketing_page.html' ,
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\defaulttags.py", line 211, in render
    nodelist.append(node.render_annotated(context))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 106, in render
    url = self.url(context)
          ^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 103, in url
    return self.handle_simple(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 118, in handle_simple
    return staticfiles_storage.url(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 147, in url
    return self._url(self.stored_name, name, force)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 126, in _url
    hashed_name = hashed_name_func(*args)
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 417, in stored_name
    raise ValueError("Missing staticfiles manifest entry for '%s'" % clean_name)
ValueError: Missing staticfiles manifest entry for '/assets/profile.jpg'
"GET /marketing_page/ HTTP/1.1" 500 145
Internal Server Error: /marketing_page/
Traceback (most recent call last):
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\Downloads\snape\SocialMediaAnalysisPlatform\snape\myproject\main\views.py", line 658, in marketing_page
    return render(request, 'main/marketing_page.html' ,
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\defaulttags.py", line 211, in render
    nodelist.append(node.render_annotated(context))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 106, in render
    url = self.url(context)
          ^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 103, in url
    return self.handle_simple(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 118, in handle_simple
    return staticfiles_storage.url(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 147, in url
    return self._url(self.stored_name, name, force)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 126, in _url
    hashed_name = hashed_name_func(*args)
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 417, in stored_name
    raise ValueError("Missing staticfiles manifest entry for '%s'" % clean_name)
ValueError: Missing staticfiles manifest entry for '/assets/profile.jpg'
"GET /marketing_page/ HTTP/1.1" 500 145
Internal Server Error: /marketing_page/
Traceback (most recent call last):
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\Downloads\snape\SocialMediaAnalysisPlatform\snape\myproject\main\views.py", line 658, in marketing_page
    return render(request, 'main/marketing_page.html' ,
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 162, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\defaulttags.py", line 211, in render
    nodelist.append(node.render_annotated(context))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 106, in render
    url = self.url(context)
          ^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 103, in url
    return self.handle_simple(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\templatetags\static.py", line 118, in handle_simple
    return staticfiles_storage.url(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 147, in url
    return self._url(self.stored_name, name, force)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 126, in _url
    hashed_name = hashed_name_func(*args)
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Welcome\AppData\Local\Programs\Python\Python311\Lib\site-packages\django\contrib\staticfiles\storage.py", line 417, in stored_name
    raise ValueError("Missing staticfiles manifest entry for '%s'" % clean_name)
ValueError: Missing staticfiles manifest entry for '/assets/profile.jpg'
"GET /marketing_page/ HTTP/1.1" 500 145
Not Found: /scrape/
"GET /scrape/ HTTP/1.1" 404 179
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 179
